File: fizzbuzz/fizzbuzz.go
//Copyright 2015 Nathan Green
package main

import (
	"bytes"
	"fmt"
	"os"
)

func main() {
  var b = new(bytes.Buffer)
  //for j := 1; j < 100000; j++ {
  for i := 1;  i <= 100; i++ {
    if i % 3 == 0 { b.Write([]byte("fizz")) }
    if i % 5 == 0 { b.Write([]byte("buzz")) }
    if b.Len() == 0 { fmt.Fprintf(b, "%d", i) }
    b.WriteByte('\n')
    b.WriteTo(os.Stdout)
    b.Reset()
  }
  //}
}

File: fizzbuzz/fizzbuzz.py
#!/usr/bin/env python
# Copyright 2015 Nathan Green

for i in range(1, 101):
    s = ''
    if i % 3 == 0: s += 'Fizz'
    if i % 5 == 0: s += 'Buzz'
    if len(s) == 0: s += str(i)
    print s
